experimental/python-flask/image/Dockerfile-stack

```diff
@@ -0,0 +1,45 @@
+FROM python:3.7
+
+RUN pip install --user pipenv
+ENV PATH=/root/.local/bin:$PATH
+
+ENV APPSODY_MOUNTS=/:/project/userapp
+ENV APPSODY_DEPS=/project/deps
+
+ENV APPSODY_WATCH_DIR=/project/userapp
+ENV APPSODY_WATCH_IGNORE_DIR=/project/userapp/dependencies
+ENV APPSODY_WATCH_REGEX="^.*.py$"
+
+ENV APPSODY_INSTALL="cd /project/userapp;pipenv lock -r > requirements.txt;python -m pip install -r requirements.txt -t /project/deps;python -m pip install ptvsd -t /project/deps;mkdir -p /project/userapp/dependencies;cp -rf /project/deps/* /project/userapp/dependencies"
```

Comment:
I think the copy command should be called in `appsody init` rather than `APPSODY_INSTALL` @neeraj-laad please correct me if I am wrong?

Reply:
You shouldn't copy the dependencies in APPSODY_INSTALL as it is used every time the user runs `appsody run`, `build`, `deploy` so it slows it down. You also already have it in the appsody-init script so I think it is fine to remove it here